.github/workflows/cargo_flake.yml

```diff
@@ -17,8 +17,11 @@ env:
 
 jobs:
   hack:
-    runs-on: [self-hosted, linux, x64, general]
     name: Cargo Flake
+    runs-on: [self-hosted, linux, x64, general]
+    strategy:
+      matrix:
+        target: [default, benches, metrics-benches, remap-benches]
```

Comment:
Should we add the other bench features here?

Reply:
Done in https://github.com/timberio/vector/pull/7776/commits/4924152abaa771df377d43db264528e557469098